docs/references/javascript/user/user.mdx

```diff
@@ -58,54 +58,149 @@ The ClerkJS SDK provides some helper [methods](#methods) on the `User` object to
 ```
 
-Updates the user's attributes. Use this method to save information you collected about the user.
-
 #### `UpdateUserParams`
 
 All props below are optional.
 
 | Name | Type | Description |
 | --- | --- | --- |
 | `username` | `string` | The user's username. |
-| `password` (deprecated) | `string` | The user's password.<br />This property is deprecated. This will be removed in the next major version (V5). Please use [`updatePassword(params)`](/docs/references/javascript/user/password-management#update-password) instead. |
 | `firstName` | `string` | The user's first name. |
 | `lastName` | `string` | The user's last name. |
 | `primaryEmailAddressId` | `string` | The unique identifier for the [`EmailAddress`][email-ref] that the user has set as primary. |
 | `primaryPhoneNumberId` | `string` | The unique identifier for the [`PhoneNumber`][phone-ref] that the user has set as primary. |
 | `primaryWeb3WalletId` | `string` | The unique identifier for the [`Web3Wallet`][web3-ref] that the user signed up with. |
 | `unsafeMetadata` | `{[string]: any} \| null` | Metadata that can be read and set from the Frontend API. One common use case for this attribute is to implement custom fields that will be attached to the `User` object.<br />Please note that there is also an `unsafeMetadata` attribute in the [`SignUp`](/docs/references/javascript/sign-up/sign-up) object. The value of that field will be automatically copied to the user's unsafe metadata once the sign up is complete. |
 
+#### `update()` returns
+
+| Type | Description |
+| --- | --- |
+| <code>Promise\<User\></code> | This method returns a `Promise` which resolves with the current `User` object. |
+
+#### `update()` example
+
+In the following example, the user's first name is updated to "Test".
+
+<InjectKeys>
+
+<CodeBlockTabs type="npm-script" options={['NPM Module', 'window.Clerk']}>
+  ```typescript
+  import Clerk from '@clerk/clerk-js';
+
+  // Initialize Clerk with your Clerk publishable key
+  const clerk = new Clerk(`{{pub_key}}`);
+  await clerk.load();
+  window.Clerk = clerk;
+
+  await clerk.user.update({ firstName: "Test" })
+  ```
+
+  ```html
+  <script>
+    // Get your publishable key and frontend API URL from the Clerk Dashboard
+    const clerkPublishableKey = `{{pub_key}}`;
+    const frontendApi = '[your-domain].clerk.accounts.dev';
+    const version = '@latest'; // Set to appropriate version
+
+    // Creates asynchronous script
+    const script = document.createElement('script');
+    script.setAttribute('data-clerk-frontend-api', frontendApi);
+    script.setAttribute('data-clerk-publishable-key', clerkPublishableKey);
+    script.async = true;
+    script.src = `https://${frontendApi}/npm/@clerk/clerk-js${version}/dist/clerk.browser.js`;
+
+    // Adds listener to initialize ClerkJS after it's loaded
+    script.addEventListener('load', async function () {
+      await window.Clerk.load();
+
+      await window.Clerk.user.update({ firstName: "Test" })
```

Comment:
I don't think this either of these examples will work unless you have a logged in user - may be worth noting this somewhere, or adding logic to sign in a user so the examples stand on their own.